Project-II-Calculator/calculator/src/components/DisplayComponents/Display.css

```diff
@@ -0,0 +1,55 @@
+.calculator {
+  display: flex;
+  flex-direction: column-reverse;
+}
+
```

Comment:
Try to focus more on the width instead of the height. The height will auto generate if you just focus on the width. That's what you mainly have control over. Really strive for keeping track of how much room each component needs to take up in its parent div.